docs/tracking.md

```diff
@@ -0,0 +1,192 @@
+
+In the above `$THE_MUPPET_SHOW` will be a hash value that acts as a stable identifier for the
+project in the network. `$KERMIT` refers to the unique peer identifier for Kermit's device. And
+`$KERMIT_IDENTITY` refers to Kermit's identity document that has metadata such as his handle and
+public keys[2].
+
+Kermit, ever the romantic, decides to get his beloved Miss Piggy involved in the Muppet Show and
+tells her the URN that identifies the project. She asks the network for the URN and Kermit's peer
+responds with the project. So what would Miss Piggy's monorepo look like?
+
+```
+refs
+`-- namespaces
+    |-- $THE_MUPPET_SHOW
+    |   `-- refs
+    |       |-- heads # <-- code branches owned by $MISS_PIGGY go here
+    |       |   `-- muppet-mania
+    |       |-- rad
+    |       |   |-- id
+    |       |   |-- signed_refs
+    |       |   |-- self # <-- points to the identity of $MISS_PIGGY
+    |       |   `-- ids
+    |       |       `-- $KERMIT
+    |       `-- remotes
+    |           `-- $KERMIT
+    |               |-- heads
+    |               |   `-- muppet-mania
+    |               `-- rad
+    |                   |-- id # <-- points to the identity of $KERMIT_IDENTITY
+    |                   |-- signed_refs
+    |                   |-- self
+    |                   `-- ids
+    |                       `-- $KERMIT_IDENTITY
+    |-- $KERMIT_IDENTITY
+    |   `-- refs
+    |       |-- heads
+    |       `-- rad
+    |           |-- id
+    |           |-- signed_refs
+    |           |-- self
+    |           `-- ids
+    |               `-- $KERMIT_IDENTITY
+    `-- $MISS_PIGGY_IDENTITY
+        `-- refs
+            |-- heads
+            `-- rad
+                |-- id
+                |-- signed_refs
+                |-- self
+                `-- ids
+                    `-- $MISS_PIGGY_IDENTITY
+```
+
+Again, `$MISS_PIGGY` and `$MISS_PIGGY_IDENTITY` are variables for Miss Piggy's peer identifier for
+the device she's using and the her identity document.
+
+As well as this entry of `remotes/$KERMIT`, Miss Piggy's monorepo will also gain an entry in its
+`config` file:
+
+```
+[remote "$THE_MUPPET_SHOW/$KERMIT"]
+        url = rad-p2p://$MISS_PIGGY@$KERMIT/$THE_MUPPET_SHOW.git?
+        fetch = +refs/heads/*:refs/remotes/$KERMIT/$THE_MUPPET_SHOW/*
+```
+
+The fetch refspec[5] here will seem off to you when comparing it to the above layout of a single
+monorepo, and you would be right. In this case Link only cares about the `url` and manages the fetch
+refspecs under the hood. The URL allows use to manage who we are tracking and for what URN.
+
+What we are learning from this exposition is that we add a peer as someone we track when we clone[3]
+their project. Miss Piggy is adding Kermit as a `remote`, and so she is tracking his changes. If
+Kermit announces changes and Miss Piggy's peer hears about these changes -- whether through Kermit's
+device or some other peer -- she will apply them to update her view of Kermit's remote.
```

Comment:
This section makes a bit of a leap. It implies there is an understanding of what it means to "hear about changes". It might be helpful to establish how gossip works, what kind of messages are disseminated and what reactions these trigger.

Because in my understanding hearing about a change doesn't mean it's locally available as there still needs a fetch to happen, or am I off?